src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -2945,6 +2948,10 @@ namespace SimAirServingZones {
 			if ( QActual > 0.0 ) CoolingActive = true; // determine if coil is ON
 
 		} else if ( SELECT_CASE_var == WaterCoil_SimpleHeat ) { // 'Coil:Heating:Water'
```

Comment:
and switched from ControlCompOutput to RegulaFalsi for water coils in the main air loop